eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -87,14 +87,16 @@ function ParseMavenPackage($pkg, $workingDirectory) {
   $pkgVersion = $contentXML.project.version
   $groupId = if ($contentXML.project.groupId -eq $null) { $contentXML.project.parent.groupId } else { $contentXML.project.groupId }
   $releaseNotes = ""
+  $readmeContent = ""
 
   # if it's a snapshot. return $null (as we don't want to create tags for this, but we also don't want to fail)
   if ($pkgVersion.Contains("SNAPSHOT")) {
     return $null
   }
 
-  if (@(Get-ChildItem -Path $pkg.DirectoryName -Recurse -Include "$($pkg.Basename)-changelog.md")[0]) {
-    $releaseNotes = &"${PSScriptRoot}/../Extract-ReleaseNotes.ps1" -ChangeLogLocation @(Get-ChildItem -Path $pkg.DirectoryName -Recurse -Include "$($pkg.Basename)-changelog.md")[0]
+  $changeLogLoc = @(Get-ChildItem -Path $pkg.DirectoryName -Recurse -Include "$($pkg.Basename)-changelog.md")[0]
```

Comment:
@scbedd given this is different from what you did in your quick fix https://github.com/Azure/azure-sdk-for-java/pull/11385/files can you please take a close look to ensure this sync is correct?

Reply:
Sure.

Reply:
After flipping back and forth between the two checkins, it definitely looks right to me.